packages/emotion-server/src/inline.js

```diff
@@ -0,0 +1,65 @@
+import { inserted, registered, names } from 'emotion'
+
+function toTag(ids, thing) {
```

Comment:
Should this just accept an array as the second argument and have the function return an object with the updated `keys` and tag?

Reply:
Yeah, that's probably a better idea.